packages/collector/test/announceCycle/unannounced_test.js

```diff
@@ -213,6 +213,71 @@ describe('unannounced state', () => {
         }
       });
     });
+    it('should apply the configuration to ignore specified endpoint for a single package', done => {
+      prepareAnnounceResponse({
+        tracing: {
+          'ignore-endpoints': {
+            redis: 'get'
+          }
+        }
+      });
+      unannouncedState.enter({
+        transitionTo: () => {
+          expect(agentOptsStub.config).to.deep.equal({
+            tracing: {
+              ignoreEndpoints: {
+                redis: ['get']
+              }
+            }
+          });
+          done();
+        }
+      });
+    });
+    it('should apply the configuration to ignore multiple endpoints for a single package', done => {
+      prepareAnnounceResponse({
+        tracing: {
+          'ignore-endpoints': {
+            redis: 'TYPE|GET'
```

Comment:
qs: Why is the agent notation `redis: 'get|type'` and the in code config for the tracer is `redis: ['get', 'type']`? Is Team Java doing that as well?

Reply:
The agent notation `redis: 'get|type'` is designed due to a restriction from Team Java. Since the agent configuration is shared, it must remain consistent, so we adopt the same pipe notation.

For tracer-specific configurations, using an array like ['get', 'type'] or a string is more convenient in JavaScript, but we can discuss and adjust this if necessary.